test/hotspot/jtreg/compiler/vectorization/runner/BasicShortOpTest.java

```diff
@@ -216,9 +216,8 @@ public short[] vectorSignedShiftRight() {
     @IR(failOn = {IRNode.STORE_VECTOR})
     public short[] vectorMin() {
         short[] res = new short[SIZE];
-        int val = 65536;
         for (int i = 0; i < SIZE; i++) {
-            res[i] = (short) Math.min(a[i], val);
+            res[i] = (short) Math.min(a[i], b[i]);
```

Comment:
I guess without this change, this collapses to a constant which enables vectorization which was not expected before?

Reply:
Yeah, exactly - since `65536` is larger than the maximum short value, with this patch it can optimize the MinI node away entirely. I changed it to use the `b` array, which is what the `vectorMax` test case below uses.